src/types/locatorApi.ts

```diff
@@ -52,34 +45,33 @@ export interface Container {
   interiorColour?: string;
   notes?: string;
   materials?: MaterialWithCategory[];
-}
-
-export interface OrganicStreamContainer extends Container {
   cost?: number;
 }
 
-export interface ResidualScheme {
-  name: string;
-  containers: Container[];
-}
-
-export interface OrganicScheme {
-  name: string;
-  containers: OrganicStreamContainer[];
+export enum PROPERTY_TYPE {
```

Comment:
@DanWebb the property keys are localised, so `cy` requests get Welsh names back 😬

Reply:
Thanks for the heads up, I'll handle this when I test adding the header https://linear.app/etch/issue/WRAP-282/send-the-accept-language-header-in-locator-api-requests